packages/stable/src/api/files/filesMultipartUploadSessionApi.ts

```diff
@@ -22,7 +22,7 @@ export class FilesMultipartUploadSessionAPI extends BaseResourceAPI<FileInfo> {
     [, , , this.multiPartFileUploadResponse] = args;
 
     this.uploadedUrls = this.multiPartFileUploadResponse.uploadUrls.map(
-      () => false
+      (_) => false
     );
     this.finished = false;
   }
```

Comment:
Error text on line 36 should be:
   throw Error('you are uploading empty content');

Error text on line 42 should be:
    throw Error('part number is outside allowed range 0 to 249');

Ideally the numbers there should in stead of being hardcoded be taken from the constants defined in FilesApi.limits, using the limit - 1.